lib/parallel_tests/tasks.rb

```diff
@@ -102,7 +102,11 @@ def parse_args(args)
   # just load the schema (good for integration server <-> no development db)
   desc "load dumped schema for test databases via db:schema:load --> parallel:load_schema[num_cpus]"
   task :load_schema, :count do |t,args|
-    command = "rake db:schema:load RAILS_ENV=#{ParallelTests::Tasks.rails_env}"
+    if Gem::Version.new(Rails.version) > Gem::Version.new('4.2.0')
+      command = "rake db:purge db:schema:load RAILS_ENV=#{ParallelTests::Tasks.rails_env}"
```

Comment:
why does this need to purge / why is "doing that rails does" not enough ?
maybe the app should overwrite "db:schema:load" to deal with foreign keys ?